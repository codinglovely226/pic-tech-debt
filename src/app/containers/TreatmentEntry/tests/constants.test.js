import {
  RESET_REFERRALS_VIEW,
  RESET_TREATMENTS_VIEW,
  UPDATE_REFERRALS,
  UPDATE_REFERRALS_VIEW_REFEREE,
  UPDATE_REFERRALS_VIEW_REFEREE_SITE,
  UPDATE_REFERRALS_VIEW_SPECIALTY,
  UPDATE_TREATMENTS,
  UPDATE_TREATMENTS_VIEW_CONTENT,
  UPDATE_TREATMENTS_VIEW_DISPOSITION,
  UPDATE_TREATMENTS_VIEW_TREATMENTS,
} from '../constants';

describe('Treatments action constants', () => {
  it('RESET_REFERRALS_VIEW', () => {
    expect(RESET_REFERRALS_VIEW).toMatchSnapshot();
  });
  it('RESET_TREATMENTS_VIEW', () => {
    expect(RESET_TREATMENTS_VIEW).toMatchSnapshot();
  });
  it('UPDATE_REFERRALS', () => {
    expect(UPDATE_REFERRALS).toMatchSnapshot();
  });
  it('UPDATE_REFERRALS_VIEW_REFEREE', () => {
    expect(UPDATE_REFERRALS_VIEW_REFEREE).toMatchSnapshot();
  });
  it('UPDATE_REFERRALS_VIEW_REFEREE_SITE', () => {
    expect(UPDATE_REFERRALS_VIEW_REFEREE_SITE).toMatchSnapshot();
  });
  it('UPDATE_REFERRALS_VIEW_SPECIALTY', () => {
    expect(UPDATE_REFERRALS_VIEW_SPECIALTY).toMatchSnapshot();
  });
  it('UPDATE_TREATMENTS', () => {
    expect(UPDATE_TREATMENTS).toMatchSnapshot();
  });
  it('UPDATE_TREATMENTS_VIEW_CONTENT', () => {
    expect(UPDATE_TREATMENTS_VIEW_CONTENT).toMatchSnapshot();
  });
  it('UPDATE_TREATMENTS_VIEW_DISPOSITION', () => {
    expect(UPDATE_TREATMENTS_VIEW_DISPOSITION).toMatchSnapshot();
  });
  it('UPDATE_TREATMENTS_VIEW_TREATMENTS', () => {
    expect(UPDATE_TREATMENTS_VIEW_TREATMENTS).toMatchSnapshot();
  });
});
