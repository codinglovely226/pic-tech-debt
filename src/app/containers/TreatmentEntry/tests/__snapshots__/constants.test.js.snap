exports[`Treatments action constants RESET_REFERRALS_VIEW 1`] = `"pic/TreatmentEntry/RESET_REFERRALS_VIEW"`;

exports[`Treatments action constants RESET_TREATMENTS_VIEW 1`] = `"pic/TreatmentEntry/RESET_TREATMENTS_VIEW"`;

exports[`Treatments action constants UPDATE_REFERRALS 1`] = `"pic/TreatmentEntry/UPDATE_REFERRALS"`;

exports[`Treatments action constants UPDATE_REFERRALS_VIEW_REFEREE 1`] = `"pic/TreatmentEntry/UPDATE_REFERRALS_VIEW_REFEREE"`;

exports[`Treatments action constants UPDATE_REFERRALS_VIEW_REFEREE_SITE 1`] = `"pic/TreatmentEntry/UPDATE_REFERRALS_VIEW_REFEREE_SITE"`;

exports[`Treatments action constants UPDATE_REFERRALS_VIEW_SPECIALTY 1`] = `"pic/TreatmentEntry/UPDATE_REFERRALS_VIEW_SPECIALTY"`;

exports[`Treatments action constants UPDATE_TREATMENTS 1`] = `"pic/TreatmentEntry/UPDATE_TREATMENTS"`;

exports[`Treatments action constants UPDATE_TREATMENTS_VIEW_CONTENT 1`] = `"pic/TreatmentEntry/UPDATE_TREATMENTS_VIEW_CONTENT"`;

exports[`Treatments action constants UPDATE_TREATMENTS_VIEW_DISPOSITION 1`] = `"pic/TreatmentEntry/UPDATE_TREATMENTS_VIEW_DISPOSITION"`;

exports[`Treatments action constants UPDATE_TREATMENTS_VIEW_TREATMENTS 1`] = `"pic/TreatmentEntry/UPDATE_TREATMENTS_VIEW_TREATMENTS"`;
